src/AppInstallerCommonCore/Public/winget/UserSettings.h

```diff
@@ -81,10 +83,21 @@ namespace AppInstaller::Settings
             static constexpr value_t DefaultValue = _default_; \
             static constexpr std::string_view Path = _path_; \
             static std::optional<value_t> Validate(const json_t& value); \
+            _extension_ \
         }
 
+#define SETTINGMAPPING_SPECIALIZATION(_setting_, _json_, _value_, _default_, _path_) \
+        SETTINGMAPPING_SPECIALIZATION_EXTEND(_setting_, _json_, _value_, _default_, _path_, )
+
+#define SETTINGMAPPING_SPECIALIZATION_POLICY(_setting_, _json_, _value_, _default_, _path_, _valuePolicy_) \
+        SETTINGMAPPING_SPECIALIZATION_EXTEND(_setting_, _json_, _value_, _default_, _path_, \
+            static constexpr ValuePolicy Policy = _valuePolicy_; \
+            using policy_t = decltype(std::declval<GroupPolicy>().GetValue<Policy>())::value_type; \
```

Comment:
`declval` should be added to https://github.com/microsoft/terminal/blob/main/.github/actions/spelling/dictionary/apis.txt